experimental/dds/sequence-deprecated/package.json

```diff
@@ -77,12 +115,14 @@
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.34.0",
 		"@fluidframework/eslint-config-fluid": "^3.5.0",
+		"@fluidframework/mocha-test-setup": "^1.3.7",
```

Comment:
`workspace:~` instead of `^1.3.7`, right?

Reply:
Looks like a bad merge or a mistake on my part.  It's now '@fluid-internal/mocha-test-setup'.  (Matt is cleaning up our use of @scopes.)